CHANGELOG.md

```diff
@@ -69,6 +69,11 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added support for the TZ environment variable (setting timezones ex.
   `"Europe/Stockholm"`) through the tzdata package. (#20)
 
+- Removed `internal/httputils`, which was moved to `wharf-core/pkg/cacertutil`.
+  (#28)
+
+- Added dependency on `wharf-core/pkg/cacertutil`. (#28)
```

Comment:
nit: Missing full package name. And IMO you can omit these package dependencies. We already have a dedicated release note for the `github.com/iver-wharf/wharf-core` Go module.

Reply:
Removed `Added dependency` entry in 0de5fbc.